lib/commands/create.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
                 return confirm({
                     type: type,
                     localDir: localDir,
-                    branch: tag,
+                    branch: branch || tag,
```

Comment:
`branch || tag` defaults to `master` instead of the highest tag.

Reply:
Should `DefaultBranch` now be removed from the constants?

Reply:
granted